openstack/keystone/values.yaml

```diff
@@ -464,3 +464,14 @@ sapcc_rate_limit:
   enabled: false
   persistence:
     enabled: false
+
+2fa:
+  enabled: false
+  replicaCount: 3
+  pollInterval: 1m #how often do we list projects
```

Comment:
`pollInterval: 1m` means that the proxy will list projects ~1 time per minute. Do i understand correctly, that N replicas will make ~N requests per minute together?

Reply:
yes each replica will call `/v3/projects?tag=xs:2fa` every `pollInterval` to maintain and a fresh local cache of auth scopes that need 2fa enforcement.
We can extend this to 5 minutes of you have concerns. This means changing the access policies in the dashboard may take up to `pollInterval` to take effect.